python/cudf/cudf/_lib/string_casting.pyx

```diff
@@ -502,14 +502,13 @@ def _from_booleans(
     return Column.from_unique_ptr(move(c_result))
 
 
-def from_booleans(Column input_col, **kwargs):
-
+def from_booleans(Column input_col):
     return _from_booleans(input_col)
 
 
 def int2timestamp(
         Column input_col,
-        **kwargs):
+        format):
```

Comment:
```suggestion
        format="%Y-%m-%dT%H:%M:%SZ"):
```

Should we have a `format` default parameter?

Reply:
My preference would be for callers to be explicit about what kind of formatting they want, especially given that this is an internal API. I think `"%Y-%m-%dT%H:%M:%SZ"` is too specific to be the default.